.github/workflows/ssr-docker.yml

```diff
@@ -0,0 +1,114 @@
+name: SSR Pipeline
+
+on:
+  push:
+    branches: 
+      - master
+    tags:
+      - 'v*'
+  pull_request:
+    branches: [ master, develop ]
+    types: [ opened, closed ]
+
+env:
+  PR_NUMBER: ${{ github.event.pull_request.number }}
+  ENV: "stg"
+  ARGOCD-FILE: "values-staging.yaml"
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - if: github.ref != 'refs/heads/develop'
+        uses: actions/checkout@v2
+        with:
+          ref: develop
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 100
+
+      - if: github.ref != 'refs/heads/develop'
+        run: git branch develop origin/develop
+      
+      - uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-${{ hashFiles('**/package-lock.json') }}
```

Comment:
why are we caching this if we do not do anything with the cache?

Reply:
I have the same question here .Do we need this cache at all ?

Reply:
Well we can use it but then we should use it to skip some steps. I don't see a reason here though. So lets remove it for now.

Reply:
Sure , will remove it for now